packages/rancher-alerting/rancher-prom2teams/generated-changes/patch/values.yaml.patch

```diff
@@ -36,12 +36,12 @@
    # readOnlyRootFilesystem is a flag to enable readOnlyRootFilesystem for the Hazelcast security context
    readOnlyRootFilesystem: true
 +
-+nodeSelector:
-+  kubernetes.io/os: linux
++## Node labels for pod assignment
++## Ref: https://kubernetes.io/docs/user-guide/node-selection/
++##
++nodeSelector: {}
++
++## List of node taints to tolerate (requires Kubernetes >= 1.6)
++tolerations: []
 +
-+tolerations:
-+  - key: cattle.io/os
-+    operator: "Equal"
-+    value: "linux"
-+    effect: NoSchedule
 +affinity: {}
```

Comment:
already has support for using the helper template to auto-insert these tolerations and nodeSelectors, not sure why it was added twice